react/react1/week2/to-do-list/src/components/to-do-list.jsx

```diff
@@ -0,0 +1,117 @@
+import ListHeader from "./to-do-header";
+import "./css/task-item.css";
+import { useState } from "react";
+
+const myList = [
```

Comment:
now it's time to separate this data in another file, so your app won't get clunky